base_local_planner: "squirrel_navigation/LocalPlanner"
base_global_planner: "squirrel_navigation/GlobalPlanner"
planner_frequency: 1.25
planner_patience: 3.0
recovery_behavior_enabled: false
controller_frequency: 15.0
controller_patience: 5.0
shutdown_costmaps: false